### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by mubix**
not sure how this got in there

### CODE REVIEW

**Comment by bcook-r7** [comment]
There's more than one!

**Comment by bcook-r7** [comment]
```
msf post(exec_powershell) > set script ls
script => ls
msf post(exec_powershell) > run

[+] Compressed size: 652
[*] #< CLIXML


    Directory: E:\metasploit-framework\mytest


Mode                LastWriteTime     Length Name
----                -------------     ------ ----
d----          3/6/2013   9:22 PM            samples
d----         3/24/2015   3:42 PM            test
d----          4/8/2015  11:53 AM            tmp
...
<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><Obj S="progress" RefId="0"><TN RefId="0"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing m
[+] Finished!
[*] Post module execution completed
```

**Comment by wvu-r7** [comment]
dat store, yo

**Comment by bcook-r7** [comment]
4b27142, 0784370

File: modules/post/windows/manage/exec_powershell.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run
     raise "Powershell not available" if ! have_powershell?
 
     # Preprocess the Powershell::Script object with substitions from Exploit::Powershell
-    script = make_subs(read_script(datstore['SCRIPT']),process_subs(datstore['SUBSTITUTIONS']))
+    script = make_subs(read_script(datstore['SCRIPT']),process_subs(datastore['SUBSTITUTIONS']))
```
Comment (FireFart) [inline modules/post/windows/manage/exec_powershell.rb]: @mubix @bcook-r7 there is another `datstore` typo in this line lol

Comment (wvu-r7) [inline modules/post/windows/manage/exec_powershell.rb]: Fixed in https://github.com/rapid7/metasploit-framework/commit/0784370b9879ce0f2eeef496df11f2aa672fbaa3.

**Pull request merged**